translate.js

```diff
@@ -48,6 +49,14 @@ function translateJsonCompilerOutput (output) {
           'sourceMap': contractInput['srcmapRuntime']
         },
         'methodIdentifiers': contractInput['functionHashes'],
+        'gasEstimates': {
+          'creation': {
+            'dataCost': gasEstimates['creation'][1],
+            'executionCost': gasEstimates['creation'][0]
+          },
+          'internal': gasEstimates['internal'],
+          'external': gasEstimates['external']
```

Comment:
The values are different between the old and the new JSON.

The new JSON has the gas values as string and also can contain `"infinite"`.

The old JSON uses `null` for infinite and anything which exceeds a Javascript number (>53 bits).